src/pages/data_submission/NIHDataManagement.jsx

```diff
@@ -0,0 +1,216 @@
+                  />
+                  <FormField
+                    id='consentProcessDidNotAddressFutureUseOrBroadSharing'
+                    defaultValue={formData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessDidNotAddressFutureUseOrBroadSharing)}
+                    type={FormFieldTypes.CHECKBOX}
+                    onChange={onAlternativeDataSharingPlanReasonsChange}
+                    toggleText='The consent process did not specifically address future use or broad data sharing for samples collected after January 25, 2015'
+                  />
+                  <FormField
+                    id='consentProcessInadequatelyAddressesRisk'
+                    defaultValue={formData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessInadequatelyAddressesRisk)}
+                    type={FormFieldTypes.CHECKBOX}
+                    onChange={onAlternativeDataSharingPlanReasonsChange}
+                    toggleText='The consent processes inadequately addresses risks related to future use or broad data sharing for samples collected after January 25, 2015'
+                  />
+                  <FormField
+                    id='consentProcessPrecludesFutureUseOrBroadSharing'
+                    defaultValue={formData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.consentProcessPrecludesFutureUseOrBroadSharing)}
+                    type={FormFieldTypes.CHECKBOX}
+                    onChange={onAlternativeDataSharingPlanReasonsChange}
+                    toggleText='The consent specifically precludes future use or broad sharing (including a statement that use of data will be limited to the original researchers)'
+                  />
+                  <FormField
+                    id='otherInformedConsentLimitationsOrConcerns'
+                    defaultValue={formData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherInformedConsentLimitationsOrConcerns)}
+                    type={FormFieldTypes.CHECKBOX}
+                    onChange={onAlternativeDataSharingPlanReasonsChange}
+                    toggleText='Other informed consent limitations or concerns'
+                  />
+                  <FormField
+                    id='otherReasonForRequest'
+                    defaultValue={formData?.alternativeDataSharingPlanReasons?.includes(alternativeDataSharingPlanReasonValues.otherReasonForRequest)}
+                    type={FormFieldTypes.CHECKBOX}
+                    onChange={onAlternativeDataSharingPlanReasonsChange}
+                    toggleText='Other'
+                  />
+                </div>
+              )}
+              <FormField // todo: other reason for request
+                id='alternativeDataSharingPlanExplanation'
```

Comment:
nit: The comment here looks like it was related to the `isRendered` condition. I'm not sure what it meant in the first place, and less sure what it means here.